docs/features/file_formats.md

```diff
@@ -1 +1,69 @@
-# Under construction
+# Input and Output file formats
+
```

Comment:
General comment : maybe we can show how "easy" it is to add a new file format (to suggest people to participate).